automation/UNIX/test-lnt.sh

```diff
@@ -0,0 +1,36 @@
+# Test clang on UNIX using Visual Studio Team Services
+
+set -ue
+set -o pipefail
+set -x
+
+if [ -z "$LNT" ]; then
+  exit 0;
+fi
+
+CLANG=${LLVM_OBJ_DIR}/bin/clang
+RESULT_LOG="${LNT_RESULTS_DIR}/result.log"
+
+
+if [ ! -e "$CLANG" ]; then
+  echo "clang compiler not found at $CLANG"
+  exit 1
+fi
+
+"$LNT_SCRIPT" runtest nt --sandbox "$LNT_RESULTS_DIR" --no-timestamp \
```

Comment:
I'd abandon the `--no-timestamp` argument so that it versions each benchmark build folder. I don't think want prior build artefacts skewing test results. If you want caching, there is definitely a way of making that work with cmake, for the `nt` suite I'm not so sure.